dockerfiles/therock_tester/no_rocm_image_ubuntu24_04.Dockerfile

```
@@ -0,0 +1,44 @@
+# This Docker image is used for TheRock builds and tests, providing a clean ROCm-less container
+
+FROM ubuntu:24.04
+
+RUN apt update && apt install sudo -y
+
+# Create tester user with sudo privileges
+RUN useradd -ms /bin/bash tester && \
+    usermod -aG sudo tester
```

Can be combined into a single command:
```suggestion
RUN useradd -m -s /bin/bash -U -G sudo tester
```